### PR TITLE
[flutter_tools] increase windows tool_integration_tests shard

### LINKED ISSUE #95584: Windows tool_integration_tests_1_5 is 3.00% flaky
The post-submit test builder `Windows tool_integration_tests_1_5` had a flaky ratio 3.00% for the past (up to) 100 commits, which is above our 2.00% threshold.

One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20tool_integration_tests_1_5/2443
Commit: https://github.com/flutter/flutter/commit/28d53d36964d37d2a0ea1f810c57e3178a81de50

Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20tool_integration_tests_1_5/2443
https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20tool_integration_tests_1_5/2434
https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20tool_integration_tests_1_5/2427

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Windows%20tool_integration_tests_1_5

Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fixing-flaky-tests to fix the flakiness and enable the test back after validating the fix (internal dashboard to validate: go/flutter_test_flakiness).

### PR DESCRIPTION
**Pull request by christopherfujino**
increase windows tool_integration_tests shards from 5 -> 6 to fix https://github.com/flutter/flutter/issues/95584 (tests are just at the timeout limit now)

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -3893,7 +3893,7 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Windows tool_integration_tests_1_5
+  - name: Windows tool_integration_tests_1_6
```
Comment (CaseyHillers) [inline .ci.yaml:3896]: This unfortunately still shows some of the pain with having to roll into the LUCI configs. We need to duplicate these builders with bringup: true, then land a separate commit to remove the label

Once we move flutter/infra to a generic builder we can get around this

Comment (christopherfujino) [reply .ci.yaml:3896]: You mean leave `tool_integration_tests_1_5` as blocking and add `tool_integration_tests_1_6` as bringup? and then after it rolls to the builder config, delete the old ones and mark the new ones as blocking?

Comment (CaseyHillers) [reply .ci.yaml:3896]: Yes. Post-submit will seem like there are duplicates, which is fine.

Comment (christopherfujino) [reply .ci.yaml:3896]: Filed https://github.com/flutter/flutter/issues/96544 to track the follow-up PR

**Comment by christopherfujino** [comment]
@CaseyHillers could you take another look?

CI failures are unrelated CDN network failures: https://github.com/flutter/flutter/issues/96550

**Review by CaseyHillers — APPROVED** [review]
flutter-lgtm
